README.md

```diff
@@ -149,13 +165,19 @@ Either a single service or multiple services can be provided as an array.
 
 The name of the service the command should be run within. If the docker-compose command would usually be `docker-compose run app test.sh` then the value would be `app`.
 
+### `push`
+
+A list of services to push in the format `service:image:tag`. If an image has been pre-built with the build step, that image will be re-tagged, otherwise docker-compose's built in push operation will be used. 
+
+Be aware that there is a race condition on tagging prebuilt images and pushing them if multiple push steps run in parallel. It's advisable to use a concurrency group in this situation.
```

Comment:
Lol, yes. Agreed. That was very developer of me :) 

I will document this better, it's just the result of the fact that you can't tag and push atomically in docker, so it's theoretically possible if you had two parallel builds tagging `my.sweet.image:blah` and pushing it at the same time that you'd be pushing the wrong image. It's possible too academic.